modules/virt-dpdk-config-map-parameters.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * virt/support/monitoring/virt-running-cluster-checkups.adoc
+
+:_content-type: REFERENCE
+[id="virt-dpdk-config-map-parameters_{context}"]
+= DPDK checkup config map parameters
+
+The following table shows the mandatory and optional parameters that you can set in the `data` stanza of the input `ConfigMap` manifest when you run a cluster DPDK readiness checkup:
+
+.DPDK checkup config map parameters
+[cols="1,1,1", options="header"]
+|====
+|Parameter
+|Description
+|Is Mandatory
+
+|`spec.timeout`
+|The time, in minutes, before the checkup fails.
+|True
+
+|`spec.param.networkAttachmentDefinitionName`
+|The name of the `NetworkAttachmentDefinition` object of the SR-IOV NICs connected.
+|True
+
+|`spec.param.trafficGeneratorRuntimeClassName`
+|The RuntimeClass resource that the traffic generator pod uses.
+|True
+
+|`spec.param.trafficGeneratorImage`
+|The container image for the traffic generator. The default value is https://quay.io/repository/kiagnose/kubevirt-dpdk-checkup-traffic-gen:latest.
```

Comment:
```suggestion
|The container image for the traffic generator. The default value is `quay.io/repository/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main`
```

Reply:
There was a mistake in the upstream documentation, fixed in https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/101.